examples/with-three-js/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "with-three-js",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "9.2.1",
+    "react": "16.12.0",
+    "react-dom": "16.12.0",
+    "react-three-fiber": "4.0.12",
+    "three": "0.112.1"
+  },
+  "devDependencies": {
+    "@zeit/next-css": "1.0.1"
```

Comment:
```suggestion
```

Reply:
Done

Reply:
I'm sorry I just pushed with another account of mine. I changed my git config and I forgot to make it correct. Thank you